init/seedDb.js

```diff
@@ -1,11 +1,15 @@
 const mongoose = require('mongoose');
 const PostModel = require('../src/post.model');
+const { posts } = require('./data/seed');
 
 mongoose.connect('mongodb://127.0.0.1:27017/howdy', {useNewUrlParser: true, useUnifiedTopology: true});
 
 async function seed() {
-  const posts = await PostModel.find().exec();
-  console.log('posts in db:', posts);
+  let fetchedPosts = await PostModel.find().exec();
+  console.log('posts in db:', fetchedPosts);
+  posts.forEach(post => new PostModel(post).save());
```

Comment:
This throws an error because the comments should be an array of objects but we pass integers. Compare `init/data/seed.js` with `src/post.model.js`.